topcoffea/modules/selection.py

```diff
@@ -335,3 +335,12 @@ def addLepCatMasks(events):
     events['is_emmm'] = ((n_e_4l==1) & (n_m_4l==3))
     events['is_mmmm'] = ((n_e_4l==0) & (n_m_4l==4))
     events['is_gr4l'] = ((n_e_4l+n_m_4l)>4)
+
+
+# Returns a mask for events with a same flavor opposite sign pair close to the Z
```

Comment:
Maybe expand on this description to say that the mask returns true if _any_ combination of 2 sfos leptons is found in the collection passed to it

Reply:
you also might want to expand the function to take an extra argument which defines the Z-peak window. This would correspond to what is currently hard coded to `10`. The extra input argument could be called `pt_window` or something.

This would make the function more flexible and also make it clearer what the function is doing when you come across it in `topeft.py`

Reply:
Thanks @Andrew42, I've made the changes you suggested.